views/index.erb

```diff
@@ -1,59 +1,327 @@
+<!-- @profile = {"about":null,"ambition":null,"codecademy":null,"dob":"1978-09-23","employment_status":"Self-employed","first_name":"TAMLYN","gender":"FEMALE","github":"tamlyn10","homepage":"","how_did_you_hear":"Course Report!","js_track_completedness":null,"last_name":"RUDOLPH","learning_experience":null,"phone_number":"07471286289","primes_sum":null,"projects":null,"rationale":null,"ruby_track_completedness":null,"twitter":"","web_track_completedness":null}
+ -->
+
+<% puts @profile %>
+
+<!-- Page Content -->
 <% if @profile %>
-  <h3><%= @profile["first_name"] %>'s Amazing Profile</h3>
 
-  <!-- TODO- Perhaps re-jig this to fit your new design / layout -->
-  <% @profile.each do |field, value| %>
-    <div class="profile-section box">
-      <h4><%= field %></h4>
-      <p class="profile-value"><%= value %></p>
+<!-- Tamlyn Intro -->
+<div class="row" style="margin-bottom: 15px;">
+                
+    <div class="col-sm-4">
+        <img class="img-responsive img-circle" src="images/tam.png" />
```

Comment:
I presume that you're on a mac here, which is case insensitive. On linux boxes this will break unless you use `images/Tam.png`